backends/advanced/src/advanced_omi_backend/utils/dropbox_audio_utils.py

```
@@ -0,0 +1,150 @@
+            dropbox_id = item["id"]  # Dropbox file ID
+
+            # Check if already processed
+            if await is_dropbox_file_already_processed(dropbox_id):
+                audio_logger.info(f"Skipping already processed file: {item['name']}")
+                skipped_count += 1
+                continue
+
+            wrapped_file = await download_and_wrap_dropbox_file(access_token, item)
+
+            # Attach Dropbox file ID
+            setattr(wrapped_file, "dropbox_file_id", dropbox_id)
```

Ruff (0.14.8)

Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)